packages/pocket-ponics-app/App.js

```diff
@@ -52,10 +54,10 @@ const AppStack = createStackNavigator({
 const App = createAppContainer(createSwitchNavigator(
 	{
 		App: AppStack,
-		Auth: AuthStack,
+		//Auth: AuthStack,
 	},
 	{
-		initialRouteName: 'Auth',
+		//initialRouteName: 'Auth',
```

Comment:
I think these were probably commented out for testing, but make sure to restore the functionality when you are done!